std/array.d

```diff
@@ -247,12 +245,20 @@ version (StdUnittest)
     assert(nAlive == 3);
 
     // No good way to ensure the GC frees this, just call the lifetime function
-    // directly. If delete wasn't deprecated, this is what delete would do.
-    _d_delarray_t(cast(void[]*)&arr, typeid(S));
+    // directly.
+    __delete(arr);
```

Comment:
Why not `destroy`?

Reply:
It failed the test when I tried. I think `destory` finalizes only the array itself, not it's contents.

Reply:
[Yeah it falls into this case:](https://github.com/dlang/druntime/blob/8d80dec1f6fc2a69a3f26fdfbc704079f8a0d661/src/object.d#L4298-L4304)
```d
/// ditto
void destroy(bool initialize = true, T)(ref T obj)
    if (!is(T == struct) && !is(T == interface) && !is(T == class) && !__traits(isStaticArray, T))
{
    static if (initialize)
        obj = T.init;
}
```